internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/hadoop/TestVenicePushJob.java

```diff
@@ -620,7 +619,6 @@ public void testKIFRepushForIncrementalPushStores() throws Exception {
     }
   }
 
-  // TODO: KIF-repush breaks when chunking is enabled. Enable data verification once VENG-9948 has been fixed.
```

Comment:
Can we add a test to verify that we get an exception when doing re-push from non-chunking to chunking store?